Domain/ModelValidationOptions.cs

```diff
@@ -0,0 +1,9 @@
+using Domain.Constants;
+
+namespace Domain;
+
+public class ModelValidationOptions
+{
+    public int NameMaxLength { get; set; } = 50;
```

Comment:
You should use IOptions https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.options.ioptions-1?view=net-8.0